doc/explanation/initialisation.rst

```diff
@@ -0,0 +1,76 @@
+
+After the initialisation is complete, you can look at the LXD configuration to confirm the setup.
+
+.. terminal::
+   :input: lxc cluster list
+   :host: micro01
+   :scroll:
+
+   +---------+------------------------------------+-----------------+--------------+----------------+-------------+--------+-------------------+
+   |  NAME   |                 URL                |      ROLES      | ARCHITECTURE | FAILURE DOMAIN | DESCRIPTION | STATE  |      MESSAGE      |
+   +---------+------------------------------------+-----------------+--------------+----------------+-------------+--------+-------------------+
+   | micro01 | https://[2001:db8:d:100::169]:8443 | database-leader | aarch64      | default        |             | ONLINE | Fully operational |
+   |         |                                    | database        |              |                |             |        |                   |
+   +---------+------------------------------------+-----------------+--------------+----------------+-------------+--------+-------------------+
+   | micro02 | https://[2001:db8:d:100::170]:8443 | database        | aarch64      | default        |             | ONLINE | Fully operational |
+   +---------+------------------------------------+-----------------+--------------+----------------+-------------+--------+-------------------+
+   | micro03 | https://[2001:db8:d:100::171]:8443 | database        | aarch64      | default        |             | ONLINE | Fully operational |
+   +---------+------------------------------------+-----------------+--------------+----------------+-------------+--------+-------------------+
+   :input: lxc storage list
+   +--------+--------+-----------------------------+---------+---------+
+   |  NAME  | DRIVER |         DESCRIPTION         | USED BY |  STATE  |
+   +--------+--------+-----------------------------+---------+---------+
+   | local  | zfs    | Local storage on ZFS        | 10      | CREATED |
+   +--------+--------+-----------------------------+---------+---------+
+   | remote | ceph   | Distributed storage on Ceph | 7       | CREATED |
+   +--------+--------+-----------------------------+---------+---------+
+   :input: lxc network list
+   +----------+----------+---------+----------------+--------------------------+-------------+---------+---------+
+   |   NAME   |   TYPE   | MANAGED |      IPV4      |           IPV6           | DESCRIPTION | USED BY |  STATE  |
+   +----------+----------+---------+----------------+--------------------------+-------------+---------+---------+
+   | UPLINK   | physical | YES     |                |                          |             | 2       | CREATED |
+   +----------+----------+---------+----------------+--------------------------+-------------+---------+---------+
+   | default  | ovn      | YES     | 192.0.2.1/24   | 2001:db8:39dc:898c::1/64 |             | 5       | CREATED |
```

Comment:
Do these IP addresses make sense here?
I replaced them with addresses from the documentation range, but that might be confusing since those are external addresses if I understand it correctly?

Reply:
Yeah, probably best to use internal addresses as OVN is likely to give rather than use the example subnets which are global.

```
+----------+----------+---------+----------------+---------------------------+-------------+---------+---------+
| default  | ovn      | YES     | 10.45.196.1/24 | fd42:67ac:35af:257a::1/64 |             | 6       | CREATED |
+----------+----------+---------+----------------+---------------------------+-------------+---------+---------+
```

Reply:
Maybe use `10.123.123.1/24` and `fd42:1234:1234:1234::1/64`?

Reply:
Thanks! Updated.